extensions/MozChangeField/lib/Post/RegressedByTypeKeyword.pm

```diff
@@ -0,0 +1,77 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# This Source Code Form is "Incompatible With Secondary Licenses", as
+# defined by the Mozilla Public License, v. 2.0.
+
+package Bugzilla::Extension::MozChangeField::Post::RegressedByTypeKeyword;
+
+use 5.10.1;
+use Moo;
+
+use Bugzilla::Keyword;
+
+sub evaluate_create {
+  my ($self, $args) = @_;
+  my $bug       = $args->{bug};
+  my $timestamp = $args->{timestamp};
+  my $dbh       = Bugzilla->dbh;
+
+  if (@{$bug->regressed_by}) {
```

Comment:
Same as the other comment, check for the inverse and `return`, and de-dent everything else.